bindings/java/iota-wallet-java/examples/ExampleProject/src/CreateAccounts.java

```diff
@@ -0,0 +1,30 @@
+// Copyright 2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import org.iota.Wallet;
+import org.iota.types.AccountHandle;
+import org.iota.types.ClientConfig;
+import org.iota.types.CoinType;
+import org.iota.types.WalletConfig;
+import org.iota.types.exceptions.WalletException;
+import org.iota.types.secret.StrongholdSecretManager;
+
+public class CreateAccounts {
+    private static final String DEFAULT_DEVELOPMENT_MNEMONIC = "hidden enroll proud copper decide negative orient asset speed work dolphin atom unhappy game cannon scheme glow kid ring core name still twist actor";
+
+    public static void main(String[] args) throws WalletException {
+        // Build the wallet.
+        Wallet wallet = new Wallet(new WalletConfig()
+                .withClientOptions(new ClientConfig().withNodes("https://api.testnet.shimmer.network"))
+                .withSecretManager(new StrongholdSecretManager("PASSWORD_FOR_ENCRYPTION", 5, "example-wallet"))
```

Comment:
Not sure if the timeout should be 5 in the examples
If you do a transaction, wait for it to be confirmed and then generate an address or do another transaction the time might have passed already and you will get an error

Reply:
okay i see so should i increase it to 10 seconds? or 15?

Reply:
I would make it null :trollface:

Reply:
ok :carlos:

Reply:
done